specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-04-01-preview/machineLearningServices.json

```diff
@@ -6112,6 +6388,243 @@
       "additionalProperties": false,
       "x-ms-discriminator-value": "ManagedIdentity"
     },
+    "ManagedNetworkDto": {
```

Comment:
>"ManagedNetworkDto": { [](http://example.com/codeflow?start=4&length=22)

Schema should have a description or title.

Please add it for all the model definitions here.

Reply:
done